assets/charts/control-plane/calico/templates/cluster-role-binding.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Honestly, the best would be to have everything in `calico.yaml`, as from `wget https://docs.projectcalico.org/v3.16.4/manifests/calico.yaml`

Reply:
IMO, we should look into integrating Tigera Operator in our helm chart.

Reply:
why not ?

Reply:
Hm, perhaps this is the way to go.